es7-persistence/src/main/java/com/netflix/conductor/es7/dao/index/ElasticSearchRestDAOV7.java

```diff
@@ -341,7 +342,18 @@ private void waitForHealthyCluster() throws Exception {
         Map<String, String> params = new HashMap<>();
         params.put("wait_for_status", this.clusterHealthColor);
         params.put("timeout", "30s");
-        Request request = new Request("GET", "/_cluster/health");
+        String elasticSearchHealthUrl = EnvUtils.getSystemEnv(EnvUtils.ELASTIC_SEARCH_HEALTH_URL);
```

Comment:
I am assuming this change is so that we can pass index specific path for appliance instead of overall cluster health

Reply:
yes only conductor specific indices are checked so that we will get improvement in conductor start time.